testing/kokoro/decrypt-secrets.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright 2021 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+TESTING_ROOT=$( dirname "$DIR" )
+
+# Work from the project root.
+cd $TESTING_ROOT
+
+# Prevent it from overriding files.
+if [[ -f "kokoro/test-env.sh" ]]; then
+    echo "testing/kokoro/test-env.sh already exists. Aborting."
+    exit 1
+fi
+
+PROJECT_ID="golang-samples-tests"
+
+gcloud secrets versions access latest --secret="golang-samples-test-env" \
```

Comment:
We generally frown upon accessing "latest" because it can lead to unpredictability.

Reply:
Is that a hard rule? For samples tests, I wonder if pinning a specific version wouldn't be most likely felt when new secrets are added but then mysteriously not used.

Reply:
It's not a hard rule, but the opposite rings true. What if you upload new secrets that aren't ready yet.

Reply:
Pinned a version and added a note to the readme to bump upon adding new secrets.